packages/strapi-plugin-graphql/services/type-builder.js

```diff
@@ -216,7 +221,9 @@ module.exports = {
     const globalId = model.globalId;
     const inputName = `${_.upperFirst(toSingular(name))}Input`;
 
-    if (_.isEmpty(model.attributes)) {
+    if (_.isEmpty(model.attributes) ||
```

Comment:
Use a temp var to clarify what this condition checks


```js
const hasOnlyPrivateAttributes =  keys(model.attributes).filter(atttributeName => !isPrivateAttribute(model, attribute)).length === 0

if (isEmpty(model.attributes) || hasOnlyPrivateAttributes) { /* ... */ }
```

Reply:
Will update it.